elegant/gui/general_pose_annotator.py

```diff
@@ -0,0 +1,95 @@
+import freeimage
+import pathlib
+import pickle
+import pkg_resources
+from PyQt5 import Qt
+import numpy
+from functools import partial
```

Comment:
instead of lots of partial functions, consider making the functions below into an object that has a `self.ris_widget`...

Moreover, this would make it easy for more advanced uses of the pose annotator (e.g. loading multiple channels into the flipbook via ipython, etc.) because you could construct the object outside of the `main()` function.